examples/internal/integration/integration_test.go

```diff
@@ -278,17 +280,27 @@ func testEchoOneof2(t *testing.T, port int, apiPrefix string, contentType string
 	}
 }
 
-func testEchoBody(t *testing.T, port int, apiPrefix string) {
+func testEchoBody(t *testing.T, port int, apiPrefix string, useTrailers bool) {
 	sent := examplepb.UnannotatedSimpleMessage{Id: "example"}
 	payload, err := marshaler.Marshal(&sent)
 	if err != nil {
 		t.Fatalf("marshaler.Marshal(%#v) failed with %v; want success", payload, err)
 	}
 
 	apiURL := fmt.Sprintf("http://localhost:%d/%s/example/echo_body", port, apiPrefix)
-	resp, err := http.Post(apiURL, "", bytes.NewReader(payload))
+
+	client := &http.Client{}
+	req, err := http.NewRequest("POST", apiURL, bytes.NewReader(payload))
 	if err != nil {
-		t.Errorf("http.Post(%q) failed with %v; want success", apiURL, err)
+		t.Errorf("http.NewRequest() failed with %v; want success", err)
+		return
+	}
+	if useTrailers {
+		req.Header.Set("TE", "trailers")
+	}
+	resp, err := client.Do(req)
```

Comment:
You can skip declaration of `client`:

```suggestion
	resp, err := http.DefaultClient.Do(req)
```

Reply:
Done. Thanks. Changed in the `testABELookupNotFound` as well.